src/CppUTest/TestHarness_c.cpp

```diff
@@ -33,6 +33,7 @@
 
 extern "C"
 {
+#include <stdio.h>
```

Comment:
Direct reliance on the C standard library is not allowed in CppUTest.

Reply:
My intuition is that this feature does not belong in CppUTest, but rather in your own force include. I don't see anything in the implementation that requires it to be tightly integrated with CppUTest itself.

`asnprintf` is non-standard; its a GCC/Linux extension. CppUTest is focused on portability across old, embedded, and resource-constrained systems. `asnprintf` isn't portable and I don't see it as especially valuable for CppUTest's target use-cases.

That's just my opinion though.@basvodde, did you have thoughts?